app/components/tipper/TipForm/TipFormAdvancedOptions.tsx

```diff
@@ -151,7 +151,7 @@ export function TipFormAdvancedOptions({
       <Divider />
       <Radio.Group
         label="Onboarding Flow"
-        defaultValue="1"
+        defaultValue={OnboardingFlow.DEFAULT}
```

Comment:
Does this still work when editing a tip that has a different onboarding flow? maybe the defaultValue line should just be removed? (see the line directly below where it sets the value)

Reply:
I tested and it worked as I would expect it: I created a new tip and changed the default value for "Onboarding" from "Standard" to "Skip" (second option). I saved it. Went back to the list, and clicked "Personalize tip" to get to the edit page, the second option was selected. I think it works. I guess "defaultValue" is ignored if "value" receives a value.

Reply:
Cool, thanks for testing!